packages/config/test/unit/index_spec.js

```diff
@@ -8,7 +8,7 @@ const configUtil = require('../../lib/index')
 chai.use(sinonChai)
 const { expect } = chai
 
-describe('src/index', () => {
+describe('config/lib/index', () => {
```

Comment:
😞 sad we needed to namespace this with the package name

Reply:
Why sad? I think the motivation behind this is that when these tests fail in CI, since they're run with a lot of other unit tests, this makes it clearer where to find them.